### PR TITLE
Synchronization section: editing nits

### PR DESCRIPTION
**Pull request by kainino0x**
@kvark just a few nits. please make sure it still matches your intent.

***

### :boom: Error: write EPROTO 140222018242432:error:1407742E:SSL routines:SSL23_GET_SERVER_HELLO:tlsv1 alert protocol version:../deps/openssl/openssl/ssl/s23_clnt.c:772:
 :boom: ###

[PR Preview](https://github.com/tobie/pr-preview#pr-preview) failed to build. _(Last tried on Mar 16, 2020, 10:33 PM UTC)_.

<details>
<summary>More</summary>

PR Preview relies on a number of web services to run. There seems to be an issue with the following one:

:rotating_light: [HTML Diff Service](http://services.w3.org/htmldiff) - The HTML Diff Service is used to create HTML diffs of the spec changes suggested in a pull request.

:link: [Related URL](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fpr-preview.s3.amazonaws.com%2Fgpuweb%2Fgpuweb%2Fpull%2F607%2F7531dda.html&doc2=https%3A%2F%2Fpr-preview.s3.amazonaws.com%2Fkainino0x%2Fgpuweb%2Fpull%2F607.html)

_If you don't have enough information above to solve the error by yourself (or to understand to which web service the error is related to, if any), please [file an issue](https://github.com/tobie/pr-preview/issues/new?title=Error%20not%20surfaced%20properly&body=See%20gpuweb/gpuweb%23607.)._
</details>

### CODE REVIEW

File: spec/index.bs [hunk 1]
```diff
@@ -416,12 +416,12 @@ Issue(gpuweb/gpuweb#514): Document read-only states for depth views.
...
 For each [=subresource=] of a [=physical resource=], the usage is tracked on the [=Queue timeline=].
 
-Issue: This section will need to be redacted to support multiple queues.
+Issue: This section will need to be revised to support multiple queues.
 
-The [=Queue timeline=] could be split into a sequence of [=usage scopes=]:
-within each scope the usage of the [=subresource=] stays unchanged,
-and the implementation transitions the [=subresource=] to the new usage
-at the [=usage scope=] boundaries.
+The [=Queue timeline=] is split into a sequence of [=usage scopes=].
```
Comment (kvark) [inline spec/index.bs]: I don't think saying "is split" reflects the semantics here either. It's basically a view. We can view the queue timeline this way, it doesn't mean that the timeline *is* a sequence of scopes.

Comment (kainino0x) [reply spec/index.bs]: Revised. How's this?

**Comment by kvark** [comment]
Needs a rebase. Also, @JusSn want to have a look?

**Comment by kainino0x** [comment]
ok, *now* it's merged properly.

File: spec/index.bs [hunk 2]
```diff
@@ -414,14 +414,19 @@ Issue(gpuweb/gpuweb#514): Document read-only states for depth views.
...
+according to the type of the subresource.
 
-Issue: This section will need to be redacted to support multiple queues.
+Issue: This section will need to be revised to support multiple queues.
 
-The [=Queue timeline=] could be split into a sequence of [=usage scopes=]:
-within each scope the usage of the [=subresource=] stays unchanged,
-and the implementation transitions the [=subresource=] to the new usage
-at the [=usage scope=] boundaries.
+On the [=Queue timeline=], there is ordered sequence of [=usage scopes=].
```
Comment (JusSn) [inline spec/index.bs]:
```suggestion
On the [=Queue timeline=], there is an ordered sequence of [=usage scopes=].
```

File: spec/index.bs [hunk 3]
```diff
@@ -414,14 +414,19 @@ Issue(gpuweb/gpuweb#514): Document read-only states for depth views.
...
+Each item on the timeline is contained within exactly one scope.
+For the duration of each scope, the set of [=usage flags=] of any given
+[=subresource=] is constant.
+The [=subresource=] transitions to the new usage
+at the boundaries between [=usage scope=].
```
Comment (JusSn) [inline spec/index.bs]:
```suggestion
A [=subresource=] may transition to new usages 
at the boundaries between [=usage scope=]s.
```

**Pull request merged**